FOX/armc/sanitization.py

```diff
@@ -242,8 +244,7 @@ def _guess_param(mc: MonteCarloABC, prm: dict,
     # Update the variable parameters
     metadata = {'min': -np.inf, 'max': np.inf, 'count': 0, 'guess': True}
     _prm = mc.param.param
-    for k, v in seq:
-        unit = mc.param.metadata.loc[(k, v['param']), 'unit'].iat[0]
```

Comment:
This would fail if the respective parameter type consists exclusively of guessed parameters.